ci-operator/config/openshift/cluster-node-tuning-operator/openshift-cluster-node-tuning-operator-master.yaml

```diff
@@ -92,8 +92,8 @@ tests:
     test:
     - as: e2e-pao-mixed-cpus
       cli: latest
-      commands: CLUSTER=mcp-only make cluster-label-worker-cnf cluster-deploy-pao
-        pao-functests-mixedcpus
+      commands: "export RESERVED_CPU_SET=\"0,2\"\nexport ISOLATED_CPU_SET=\"1,3\"
+        \nCLUSTER=mcp-only make cluster-label-worker-cnf cluster-deploy-pao\npao-functests-mixedcpus\n"
```

Comment:
I'm not sure this syntax with `\n` is correct

Reply:
updated, thanks for the comment